attest/verifier/types/src/verification.rs

```diff
@@ -27,16 +27,108 @@ pub struct DcapEvidence {
     pub report_data: EnclaveReportDataContents,
 }
 
-#[derive(Clone, Oneof)]
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
 pub enum EvidenceKind {
-    #[prost(message, tag = "4")]
+    Epid(VerificationReport),
```

Comment:
Other places I've used `AttestationEvidence` with a variant of `VerificationReport`
https://github.com/mobilecoinfoundation/mobilecoin/blob/c709b85bd47b31402003f09b84a1e9a9496a3356/blockchain/types/src/block_metadata.rs#L18

We probably need to revisit the naming here and in other places to be consistent